doc/add-source.rst

```diff
@@ -25,6 +25,9 @@ Options
 
      add-source --http-header "X-API-Key: 1234"
 
+   HTTP basic login can be achieved by setting the HTTP Basic Authentication header with ``base64(user1:password1)``. Example::
```

Comment:
I think this should be "HTTP basic authentication"...

Reply:
> Can you create a ticket for this please? And I prefer this one that allows whitespace in the value field no matter the header. Just simpler. Thanks!

Hi, thanks for your feedback 👍 

- Ticket created: https://redmine.openinfosecfoundation.org/issues/4362
- HTTP BasicAuth v2 Pull Request (#273) closed

Reply:
> I think this should be "HTTP basic authentication"...

Done in https://github.com/OISF/suricata-update/pull/272/commits/ae53e775667b17cdfb2a5c8cbce0c45a0767d96d.